packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -11,10 +11,68 @@
  */
 
 .root {
-  @include input-baseline;
+  @include border-radius(var(--BorderRadius));
+
+  display: block;
+  position: relative;
+  width: 100%;
+  max-width: var(--MaxWidth);
+  margin-block: 0;
+  margin-inline: 0;
+  padding-block: var(--PaddingBlock);
+  padding-inline: var(--PaddingInline);
+  transition-property: border-color, background-color, box-shadow;
+  transition-duration: var(--TransitionDuration);
+  transition-timing-function: var(--TransitionTiming);
+  border: var(--BorderWidth) var(--BorderStyle) var(--ColorBorder);
+  background-color: var(--ColorBackground);
+  color: var(--ColorText);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
+  line-height: var(--LineHeight);
+
+  &:hover,
+  &:focus {
+    border-color: cv("blue");
```

Comment:
`var(--ColorBorderPrimaryBase)` should be equivalent here and allow for theming borders anotehr requirement from SIW team

Reply:
👍